app/services/intake_progress_calculator.rb

```diff
@@ -8,14 +8,20 @@ class IntakeProgressCalculator
   def get_progress(controller, intake)
     return 0 if controller == POSSIBLE_STEPS.first
 
+    # The DependentsController is not in the QuestionsFlow, so we hold progress at the HadDependentsController until they move on
     if controller == DependentsController
       controller = Questions::HadDependentsController
     end
 
-    current_flow = POSSIBLE_STEPS.select do |controller_step|
+    steps_for_intake = POSSIBLE_STEPS.select do |controller_step|
       controller_step.show?(intake)
     end
-    current_index = current_flow.index(controller)
-    ((current_index + 1)/current_flow.count.to_f * 100).round
+    current_index = steps_for_intake.index(controller)
+    completed_steps = current_index + 1 # Everything that the client has done
+
+    index_of_possible_steps = POSSIBLE_STEPS.index(controller)
+    possible_remaining_steps = POSSIBLE_STEPS[index_of_possible_steps + 1..-1].length # Everything the client could potentially do
+
```

Comment:
Nice! I am glad that you went ahead and made this improvement on the calculation.